src/binaryen-c.cpp

```diff
@@ -299,12 +299,13 @@ BinaryenOp BinaryenCurrentMemory(void) { return CurrentMemory; }
 BinaryenOp BinaryenGrowMemory(void) { return GrowMemory; }
 BinaryenOp BinaryenHasFeature(void) { return HasFeature; }
 
-BinaryenExpressionRef BinaryenBlock(BinaryenModuleRef module, const char* name, BinaryenExpressionRef* children, BinaryenIndex numChildren) {
+BinaryenExpressionRef BinaryenBlock(BinaryenModuleRef module, const char* name, BinaryenExpressionRef* children, BinaryenIndex numChildren, BinaryenType type) {
```

Comment:
For reference, this is where the C API breaks while JS does not, which is unfortunate of course, Though, I require this to emulate what `teeLocal` does for locals, but for globals. More precisely, I am using a typed block there that first performs a `setGlobal` followed by a `getGlobal`, which is the block's result, effectively doing the same (though not nearly thread-safe) as a `teeGlobal` instruction, if it'd exist.